CMakeLists.txt

```diff
@@ -206,6 +201,7 @@ target_include_directories(${PROJECT_NAME} PUBLIC
         $<INSTALL_INTERFACE:include>)
 
 aws_use_package(aws-c-common)
+aws_use_package(aws-c-cal)
```

Comment:
need to update README.txt so it walks through installing aws-c-cal in addition to aws-c-common